### ISSUE TITLE
Declaration API: Link up members with the interfaces they implement

### ISSUE THREAD

**Issue by ghost** [rubberduck-vba/Rubberduck#1561]
@Hosch250 had the idea of linking up members and the interfaces/interface members to implement since that would make a lot of things easier. Currently the class declarations have the supertypes/subtypes they implement but the members don't have anything like that.

**Comment by retailcoder** [comment]
Yes absolutely - this would greatly simplify finding implementations and speed up inspections that need to account for interface members.
